tests/meerkat/ops/test_groupby.py

```diff
@@ -0,0 +1,226 @@
+import meerkat
+from meerkat.datapanel import DataPanel
+
+from meerkat.ops.groupby import DataPanelGroupBy
+from meerkat import NumpyArrayColumn, TensorColumn, ListColumn
+import numpy as np
+import pytest
+
+from meerkat.ops.groupby import groupby
+
+
+# Comment for meeting 5/19: Testing group by multiple columns, 
+# single columns on list, on string.
+
+# Different columns as by: including ListColumn, NumpyArrayColumn, 
+# TensorColumn
+
+# Key index is NumpyArrays, TensorColumn
+
+# Coverage: 100% 
+
+def assertNumpyArrayEquality(arr1, arr2):
+    assert np.linalg.norm( arr1 -  arr2) < 1e-10
+
+
```

Comment:
Tests are looking good - let's add a couple tests for some multidimensional arrays (e.g. embeddings) w/ and w/o the axis kwarg